packages/store-ui/src/atoms/Slider/Slider.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+
+export type SliderProps = {
+  min: number
+  max: number
+  showValues?: boolean
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  onChange?: (value: { min: number; max: number }) => void
+}
+
+const Slider = ({
+  min,
+  max,
+  onChange,
+  testId,
+  showValues = true,
+}: SliderProps) => {
+  const [minVal, setMinVal] = useState(min)
```

Comment:
LGTM, i'm just not sure if this would be simpler by using `useReducer`

Reply:
I think if we use `useReducer` it gets more complex